domino/http_request_manager.py

```diff
@@ -10,6 +10,9 @@
 from .constants import DOMINO_VERIFY_CERTIFICATE
 from .exceptions import ReloginRequiredException
 
+class _SessionInitializer:
+    def __initialize__(self, session):
+        raise NotImplementedError('Session initializers must be callable.')
```

Comment:
not sure I understand this, some python incantation

Reply:
I copied this from AuthBase. Basically, this method needs to be overridden, which we can't verify at compile time because Python